### PR TITLE
[ADI] Add /ideas/uuid endpoint

### PR DESCRIPTION
**Pull request by JeremieSamson**

### CODE REVIEW

File: src/Entity/IdeasWorkshop/Idea.php [hunk 1]
```diff
@@ -161,10 +182,10 @@ class Idea implements AuthorInterface, ReportableInterface, VisibleStatusesInter
      *
      * @Assert\Choice(
      *     callback={"AppBundle\Entity\IdeasWorkshop\IdeaStatusEnum", "toArray"},
-     *     strict=true
+     *     strict=true,
      * )
      *
-     * @SymfonySerializer\Groups({"idea_list_read", "idea_write", "idea_publish"})
```
Comment (illusionOfParadise) [inline src/Entity/IdeasWorkshop/Idea.php]: Why did you remove this group?

File: src/Entity/IdeasWorkshop/Idea.php [hunk 2]
```diff
@@ -209,7 +230,7 @@ class Idea implements AuthorInterface, ReportableInterface, VisibleStatusesInter
      *
      * @Assert\NotBlank(message="idea.description.not_blank", groups={"idea_publish"})
      *
-     * @SymfonySerializer\Groups({"idea_list_read", "idea_write", "idea_publish"})
+     * @SymfonySerializer\Groups({"idea_list_read", "idea_write"})
```
Comment (illusionOfParadise) [inline src/Entity/IdeasWorkshop/Idea.php]: Why did you remove this group?

File: src/Normalizer/IdeasWorkshopNormalizer.php [hunk 3]
```diff
@@ -27,9 +27,11 @@ public function normalize($object, $format = null, array $context = [])
     {
         $data = $this->normalizer->normalize($object, $format, $context);
 
-        if (\in_array('idea_list_read', $context['groups'])) {
-            $data['contributors_count'] = $this->ideaRepository->countIdeaContributors($object);
-            $data['comments_count'] = $this->ideaRepository->countThreadComments($object);
+        if (\in_array('idea_list_read', $context['groups']) || \in_array('idea_read', $context['groups'])) {
+            if (\in_array('idea_list_read', $context['groups'])) {
```
Comment (illusionOfParadise) [inline src/Normalizer/IdeasWorkshopNormalizer.php]: Are you sure that we don't need `contributors_count` and `comments_count` information for the idea item endpoint?

File: src/Entity/IdeasWorkshop/Idea.php (hunk 1 above)
Comment (JeremieSamson) [reply src/Entity/IdeasWorkshop/Idea.php]: merge difficulty sorry

File: src/Entity/IdeasWorkshop/Idea.php (hunk 2 above)
Comment (JeremieSamson) [reply src/Entity/IdeasWorkshop/Idea.php]: Same here

File: src/Normalizer/IdeasWorkshopNormalizer.php (hunk 3 above)
Comment (JeremieSamson) [reply src/Normalizer/IdeasWorkshopNormalizer.php]: I don't think those informations are needed based on wireframe, I'll double check with PO.

**Comment by ottaviano** [comment]
need rebase
